apidoc/Titanium/Media/Media.yml

```diff
@@ -2115,6 +2124,39 @@ properties:
     osver: {ios: {min: "11.0"}}
 
 ---
+name: CameraMediaMultipleItemsType
+summary: A media object from photo gallery when <Titanium.Media.allowMultiple> is `true` or in iOS <Titanium.Media.selectionLimit> is set.
+extends: SuccessResponse
+properties:
+  - name: success
```

Comment:
shouldn't have to document the properties already on the parent `SuccessResponse` type: success, error, code